ui/app/src/components/Search/URLDrivenSearch.tsx

```diff
@@ -209,7 +209,7 @@ export function URLDrivenSearch(props: URLDrivenSearchProps) {
       sortOrder={queryParams['sortOrder'] as string}
       currentView={currentView}
       embedded={embedded}
-      keyword={keyword}
+      keyword={Array.isArray(keyword) ? keyword.join(' ') : keyword}
```

Comment:
URLDrivenSearch set the keyword by getting the urlParam, so it may be an array of strings. Moved the logic here so the SearchUI will always get a string as the keyword